girder_annotation/girder_large_image_annotation/web_client/views/imageViewerWidget/geojs.js

```diff
@@ -402,8 +489,15 @@ var GeojsImageViewerWidgetExtension = function (viewer) {
                         this.featureLayer.deleteFeature(feature);
                     }
                 });
+                _.each(this._annotations[annotation.id].overlays, (overlay) => {
+                    const overlayLayer = this.viewer.layers().find(
+                        (layer) => layer.id() === overlay.id);
+                    this.viewer.deleteLayer(overlayLayer);
+                });
                 delete this._annotations[annotation.id];
                 delete this._featureOpacity[annotation.id];
+                this.viewer.clampBoundsY(true);
```

Comment:
I think we should reset to true if we didn't change it to false, so a downstream viewer could use whatever default they see fit if the flag is false.

Reply:
2cf5c3f